src/components/titles/index.js

```diff
@@ -102,7 +102,7 @@ function draw(gd, titleClass, options) {
     var editAttr;
     if(prop === 'title.text') editAttr = 'titleText';
     else if(prop.indexOf('axis') !== -1) editAttr = 'axisTitleText';
-    else if(prop.indexOf('colorbar' !== -1)) editAttr = 'colorbarTitleText';
+    else if(prop.indexOf('colorbar') !== -1) editAttr = 'colorbarTitleText';
```

Comment:
I was not able to replicate a bug as a result of this typo.
The typo was present since 2017 from https://github.com/plotly/plotly.js/commit/b4e26aed4e048a067338cd4895a15d7faed79e91.

The last condition would translate to 
```js
else if(prop.indexOf(true) editAttr = 'colorbarTitleText';
```
Which is obliviously not what we wanted.